web/docs/deployment/deployment-methods/wasp-deploy/railway.md

```diff
@@ -192,7 +206,21 @@ The project name is used as a base for your server and client service names on R
 - `<project-name>-client`
 - `<project-name>-server`
 
-Railway also creates a PostgreSQL database service named `Postgres`.
+Railway also creates a PostgreSQL database service named `Postgres` (the name remains the same even when using `--db-image`).
+
+#### Using a custom database image
```

Comment:
Sucks that we have all this repetition in docs. I sthere a way we could avoid it? Just link from one to another or sometihng? Maybe not?